README.md

```diff
@@ -45,7 +45,7 @@ The documentation can be found on
 
 ## Requirements (i.e. what it was designed for)
 
-- [Python](https://python.org) 2.7 or >= 3.4
+- [Python](https://python.org) >= 3.6
```

Comment:
It would be good to add a comment that previous versions supported 2.7 and it *may* still work.

Reply:
looking again, I see that we already have a statement to this effect a few lines further down:
```
In practice, pyFFTW *may* work with older versions of these dependencies, but it is not tested against them.
```

Reply:
I can add it in line as well if you like. I see that NumPy has that stated in-line